src/app/api/mux/route.ts

```diff
@@ -0,0 +1,58 @@
+import Mux, { type Asset } from "@mux/mux-node";
```

Comment:
Only `Mux.Webhooks` maybe import just import `Webhooks`?

Reply:
It looks like `Webhooks` needs an instance of `Mux` class and cannot be used imported